client/src/TerritoryStats.ts

```diff
@@ -0,0 +1,31 @@
+class TerritoryStats {
```

Comment:
Ideally the frontend would use the same model as the backend so update this to match or use the backend model directly for now.

Reply:
React does not allow the use of files outside (above?) the src/ folder if initialized as a create-react-app due to config. This is possible to correct either with a library such as lerna, rewire, or by ejecting the project and adjusting the webpack config manually. This seems above my pay grade, so I'll let you try to handle it without exploding everything https://stackoverflow.com/questions/44114436/the-create-react-app-imports-restriction-outside-of-src-directory